src/main/webapp/WEB-INF/views/cars/getAllCars.jsp

```diff
@@ -0,0 +1,32 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All cars</title>
+</head>
+<body>
+<h1><p><strong>all cars in db</strong></p></h1>
+<table border="1">
+    <thead>
+    <tr>
+        <th>Id</th>
+        <th>model</th>
+        <th>manufacturer</th>
+        <th>delete</th>
+    </tr>
+    </thead>
+    <c:forEach items="${cars}" var="car">
+        <tr>
+            <td><c:out value="${car.id}" /></td>
+            <td><c:out value="${car.model}" /></td>
+            <td><c:out value="${car.manufacturer.name}" /></td>
+            <td>
+                <a href="http://localhost:8080/cars/delete/?id=${car.id}">Delete</a>
```

Comment:
you should use ${pageContext.request.contextPath} instead of localhost:8080 in all places